File: Sc.Models/Dtos/Auth/SignUpDeclarationDto.cs
using Sc.Models.Dtos.ApproveRegistrations;

namespace Sc.Models.Dtos.Auth
{
    public class SignUpDeclarationDto
    {
        public SignUpDto SignUp { get; set; }
        public ApproveRegistrationFileDto File { get; set; }
    }
}

File: WebApplication/src/auth/dtos/sign-up/sign-up-declaration.dto.ts
import { ApproveRegistrationFileDto } from "src/approve-registrations/dtos/approve-registration-file.dto";
import { SignUpDto } from "./sign-up.dto";

export class SignUpDeclarationDto {
    file: ApproveRegistrationFileDto;
    signUp: SignUpDto = new SignUpDto();
}